Foods.yaml

```diff
@@ -6,4 +6,8 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
+<<<<<<< HEAD
```

Comment:
You'll want to get rid of these lines and fix up the merge conflict.

Reply:
Hi, I did not understand what you've told. Can you please explain a bit about how to do this?

Reply:
![image](https://user-images.githubusercontent.com/35280100/143817707-118a0c8d-fdff-474c-b896-7fa63456a408.png)

If you click the button "Files changed" you will see that your changes are probably not what you'd expect. This is because 
- You created the branch and made changes
- Someone updated the main branch with their favourite foods
- You merged main into your branch, so you were up to date with the new additions

The next step is to handle the conflict. Git doesn't know how to manually merge your line and the new line on main, so you need to tell it what it should be.

Between `<<<<<<< HEAD` and `=======` are your original changes *before* merging *main*.
Between `=======` and `>>>>>>> main` is what is currently on *main*. You'll see it isn't labelled as a change, because that is currently what is on the repository.

What you need to do is manually compare lines *10* and *12* and merge them together, such that all the new foods from others are there and your own new food is also there. This will mostly just involve adding 'pasta' to line *12* and removing the other line. 

Then you'll want to remove the lines with `<<<<<<< HEAD`, `=======` and `>>>>>>> main`.

In the end, the only additions to that file (shown in green in the 'Files changed' tab) should be "pasta". 

There is a similar issue in the second file, but this one is easier to handle because you only need to remove the extra lines Git has created.